### PR TITLE
[Task]: Move the search facet "Access level" display as radio buttons

### PR DESCRIPTION
**Pull request by Alexandra-Green**
## What this PR accomplishes

- Access level facet is now radio buttons in the grey search box and default to open

## Issue(s) addressed

- DATA-1266 & DATA-1371

## What needs review

- **Different search queries and filter options should be tried to ensure nothing breaks**.
- Access level radio buttons display datasets based on selected option
- Radio buttons appear in order of openness (open, under review, restricted)
- Dataset count/query count match "All levels" count
- Radio buttons appear on all search pages with facets and one or more datasets
- When sorting or using other facets, dataset count on radio buttons remains correct.
- Datasets and radio button selected are open by default
- Radio button translates correctly
- Radio buttons are vertical in mobile and horizontal in desktop
- Order of search box content is:
   1. query count/dataset count
   2. radio buttons
   3. facet pills

### CODE REVIEW

**Comment by KatiRG** [comment]
Different search queries and filter options should be tried to ensure nothing breaks.
    Access level radio buttons display datasets based on selected option
    Radio buttons appear in order of openness (open, under review, restricted)

- [ ]     Radio buttons and dataset count/query count match
- [ ]     Radio buttons appear on all search pages with facets and one or more datasets
- [ ]     When sorting or using other facets, dataset count on radio buttons remains correct.
- [ ]     Datasets and radio button selected are open by default
- [ ]     If open is not an option, then all levels is selected
- [ ]     Radio button translates correctly
- [ ]     Radio buttons are vertical in mobile and horizontal in desktop
- [ ]     Order of search box content is:
        - [ ] query count/dataset count
        - [ ] radio buttons
        - [ ] facet pills

File: ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po [hunk 1]
```diff
@@ -32,6 +32,14 @@ msgstr "Autre"
 msgid "Explore"
 msgstr "Explorer"
 
+#: ckanext/ontario_theme/templates/internal/snippets/access_level_radio_buttons.html:24
+msgid "Access level"
+msgstr "Niveau d'accès"
+
+#: ckanext/ontario_theme/templates/internal/snippets/access_level_radio_buttons.html:47
+msgid "All levels"
+msgstr "Tous les niveau"
```
Comment (KatiRG) [inline ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po:41]: plural niveau should have an x: niveaux

**Comment by KatiRG** [comment]
- [ ]     Different search queries and filter options should be tried to ensure nothing breaks.
- [ ]     Access level radio buttons display datasets based on selected option
- [ ]     Radio buttons appear in order of openness (open, under review, restricted)
- [ ]     Dataset count/query count matc
- [ ]     Different search queries and filter options should be tried to ensure nothing breaks.
- [ ]     Access level radio buttons display datasets based on selected option
- [ ]     Radio buttons appear in order of openness (open, under review, restricted)
- [ ]     Dataset count/query count match "All levels" count
- [ ]     Radio buttons appear on all search pages with facets and one or more datasets
- [ ]     When sorting or using other facets, dataset count on radio buttons remains correct.
- [ ]     Datasets and radio button selected are open by default
- [ ]     Radio button translates correctly
- [ ]     Radio buttons are vertical in mobile and horizontal in desktop
- [ ]     Order of search box content is:h "All levels" count
- [ ]     Radio buttons appear on all search pages with facets and one or more datasets
- [ ]     When sorting or using other facets, dataset count on radio buttons remains correct.
- [ ]     Datasets and radio button selected are open by default
- [ ]     Radio button translates correctly
- [ ]     Radio buttons are vertical in mobile and horizontal in desktop
- [ ]     Order of search box content is:
    - [ ]     query count/dataset count
    - [ ]     radio buttons
    - [ ]     facet pills

**Review by KatiRG — APPROVED** [review]

**Review by aimalkhan — APPROVED** [review]
👍

**Pull request merged**